components/builder-db/src/migration.rs

```diff
@@ -17,22 +17,15 @@ use diesel::query_dsl::RunQueryDsl;
 use diesel::sql_query;
 use error::Result;
 
-pub fn shard_setup(conn: &PgConnection, shard_id: u32) -> Result<()> {
-    debug!("Migrating shard_{:?}", shard_id);
-    sql_query(format!("CREATE SCHEMA IF NOT EXISTS shard_{}", shard_id))
-        .execute(conn)
-        .unwrap();
-    sql_query(format!("SET SEARCH_PATH TO shard_{}", shard_id))
-        .execute(conn)
-        .unwrap();
+pub fn shard_setup(conn: &PgConnection) -> Result<()> {
     // TED: We have to do this here because we rely on the shard id which isn't available in raw sql
-    sql_query(format!(
+    sql_query(
         r#"CREATE OR REPLACE FUNCTION next_id_v1(sequence_id regclass, OUT result bigint) AS $$
                 DECLARE
                     our_epoch bigint := 1409266191000;
                     seq_id bigint;
                     now_millis bigint;
-                    shard_id int := {};
+                    shard_id int := 0;
```

Comment:
Remove this line entirely, and remove the use of the shard_id in the result below?

Reply:
Ah, of course, because anything ORed with 0 is just the thing.  Will do.